react-charts/area-chart/step7.md

```diff
@@ -1,57 +1,14 @@
-Now we'll add a legend to the area chart...
+In this step you will change the area chart theme color.

```

Comment:
Let's try to remove the instances of "you will" when we can. So for example, this could read, "Change the area chart theme color."